ray-operator/test/e2eautoscaler/raycluster_autoscaler_test.go

```diff
@@ -355,3 +356,69 @@ func TestRayClusterAutoscalerMinReplicasUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestRayClusterAutoscalerV2IdleTimeout(t *testing.T) {
+	// Only test with the V2 Autoscaler
+	name := "Create a RayCluster with autoscaler v2 enabled"
+	tc := tests["Create a RayCluster with autoscaler v2 enabled"]
+
+	test := With(t)
+	g := gomega.NewWithT(t)
+
+	// Create a namespace
+	namespace := test.NewTestNamespace()
+
+	// Minimum Ray Version for custom idleTimeoutSeconds
+	idleTimeoutMinRayVersion := "2.40.0"
```

Comment:
We don't have a convention for stating the minimum requirement in tests, so you can just use the current version by `GetRayVersion()`.